src/app-layer-parser.c

```diff
@@ -1045,11 +1045,8 @@ int AppLayerParserGetStateProgressCompletionStatus(AppProto alproto,
                                                    uint8_t direction)
 {
     SCEnter();
-    int r = 0;
-    if (alproto != ALPROTO_UNKNOWN) {
-        r = alp_ctx.ctxs[FLOW_PROTO_DEFAULT][alproto].
-                    StateGetProgressCompletionStatus(direction);
-    }
```

Comment:
I prefer not to see commits that fix other commits in a PR. Can you update the original commit that made this change instead?

Reply:
Will do.